cmd/gateway-main.go

```diff
@@ -36,36 +36,30 @@ USAGE:
 FLAGS:
   {{range .VisibleFlags}}{{.}}
   {{end}}{{end}}
+BACKEND:
+  azure: Microsoft Azure Blob Storage. Default ENDPOINT is https://core.windows.net
```

Comment:
Since the final url looks like azureaccountname.blob.core.windows.net which is resolvable it is fine